src/Orleans.Core/Orleans.Core.csproj

```diff
@@ -30,12 +30,14 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsDependencyInjectionVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingVersion)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
+    <PackageReference Include="System.Buffers" Version="4.5.0" />
```

Comment:
Don't we need a variable to this version? And another one for `System.Memory`?

Reply:
yep, I have a change locally to fix that